changelog/patch-4.1.md

```diff
@@ -62,11 +69,10 @@ patch announcement
 - The key "S" now toggles the emote menu in game
 - The key "A" now plays a user avatar's animation in game
 - User in-game avatars can now use animations during carousel phases
-- Added in-game modal for displaying all possible keybinds
+- Added in-game modal for displaying all the keyboard shortcuts
 - Redesigned **Online** avatars
 - Improved **Chat** timestamp displays
-- Pokemon details shown on hover not attach to mouse instead of pokemon
-  - This should help avoid situations where you're unable to click/hover on pokemon to the right of what's being hovered
+- Pokemon details shown on hover not attach to mouse instead of pokemon. This should help avoid situations where you're unable to click/hover on pokemon to the right of what's being hovered
```

Comment:
Ahh I had a word misspelled on my original PR. If you wouldn't mind updating with this that'd be perfect. Also is there a reason we wouldn't want a sub-bullet here? Figured it let's the important part stand out a bit more.

```suggestion
- Pokemon details shown on hover now attach to mouse instead of pokemon. This should help avoid situations where you're unable to click/hover on pokemon to the right of what's being hovered
```